### DISCUSSION TITLE
You can edit a map and then play it and score will submit

### DISCUSSION THREAD

**Discussion by weajus** [ppy/osu#19160] [Q&A] [answered]
Example

![osu_2022-07-16_10-50-42](https://user-images.githubusercontent.com/50848502/179362189-6dbd3b48-309b-4e12-b0af-8331ec588083.jpg)

**Comment by bdach** [comment, marked as answer]
duplicate of https://github.com/ppy/osu/issues/11922

steps have been taken to resolve this (https://github.com/ppy/osu/pull/18978) but i don't believe that the map hash is validated server-side yet.
